06-files-input-and-output/remove_files.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+import os
+
+def find_files_in_folder(path, extension, file_max_size):
+    print(
+        "Searching for files with .{} extension in \"{}\" and maximum size of {}bytes".format(
+            extension, path, file_max_size
+        )
+    )
+
+    file_list = []
+
+    for current_folder, subfolders, file_names in os.walk(path):
+        for file_name in file_names:
+            full_path = os.path.abspath(os.path.join(current_folder, file_name))
```

Comment:
No need to call `abspath`, the `current_folder` is already the full path to the file directory. You can just use `join`, as stated in the manual page for os.path:

> To get a full path (which begins with top) to a file or directory in dirpath, do os.path.join(dirpath, name).

See [the man page](https://docs.python.org/2/library/os.html#os.walk) for more details.